composer.json

```diff
@@ -61,6 +61,9 @@
         "symfony/profiler-pack": "^1.0",
         "symfony/test-pack": "^1"
     },
+    "replace": {
+        "namshi/jose": "^7.2"
+    },
```

Comment:
This trick prevents the installation of the unmaintained `namshi/jose` library, which is indirectly required by `lexik/jwt-authentication-bundle` for backward compatibility reasons (unused unless explicitly asked via configuration).